### ISSUE TITLE
[Web] custom ops

### ISSUE THREAD

**Issue by stroyine** [microsoft/onnxruntime#15374]
### Describe the issue

Is there currently a way to register custom (self-defined) ops in onnxruntime wasm? 
Since dynamic linking is not supported, would there be any other possibility (e.g. statically link own library of custom ops during build)? 

### To reproduce

-

### ONNX Runtime Installation

Other / Unknown

### ONNX Runtime Version or Commit ID

-

### Execution Provider

WASM

**Comment by baijumeswani** [comment]
@fs-eire would you be able help answer this question?

**Comment by guschmue** [comment]
Yes, we do it this way to include some of onnxruntime-extensions in ort-web, for example tokenizers.
The registration for this is here:
https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/wasm/api.cc#L116

**Comment by stroyine** [comment]
@guschmue Could you elaborate on it a bit? I am aware that linking onnxruntime-extensions is possible out of the box. Lets say I have a few operators implemented as per guide https://onnxruntime.ai/docs/reference/operators/add-custom-op.html#create-a-library-of-custom-operators Would there be a straight-forward way to bring those operators into onnxruntime-wasm ?

**Comment by Quie7** [comment]
I've met the same question
